src/content/teams/12_formatting.mdx

```diff
@@ -0,0 +1,30 @@
+---
+name: Nix Formatting Team
+description: Maintains the official Nix formatter and standard format.
+members:
+- name: Silvan Mosberger
+  github: infinisil
+  title: Administrative leader, sponsored by Tweag and Antithesis
```

Comment:
Is there a precedent for such a sponsors remark?

Reply:
Me and @fricklerhandwerk also have that for the [docs team](https://nixos.org/community/teams/documentation/), as does @ConnorBaker from the [cuda team](https://nixos.org/community/teams/cuda/), all from Tweag